src/indicator/volatility/projectionOscillator.ts

```diff
@@ -77,7 +142,29 @@ export function projectionOscillator(
 export function defaultProjectionOscillator(
   highs: number[],
   lows: number[],
-  closings: number[]
+  closings: number[],
 ): ProjectionOscillator {
   return projectionOscillator(14, 3, highs, lows, closings);
 }
+function rng(mean: number, stdDev: number) {
+  return mean + stdDev * (Math.random() - 0.5);
+}
+
+function generateRandomNumbers(count: number, mean: number, stdDev: number) {
```

Comment:
May be the function can be named differently to reflect how you are using the mean and stdDev? I guess they are not just random number right? Also would be nice to have this in ../../helper/numArray.ts as well.